### PR TITLE
Remove artifactory refs

### LINKED ISSUE #45: Steps needed to build outside of the internal Intel network
The steps we need to take to get fully open and building outside of the internal Intel network:

- [x] curl https://ubit-artifactory-or.intel.com/artifactory/api/npm/iml-npm-repos/npm/-/npm-3.10.6.tgz > /tmp/npm-3.10.6.tgz
- [x] Remove all references to artifactory.

### PR DESCRIPTION
**Pull request by jgrund**
Remove all refs to artifactory.

In addition, stop pulling npm. It should come with node.

Partially resolves #45

### CODE REVIEW

**Comment by brianjmurrell** [comment]
I wonder if you want to rebase this on master instead of *use-lustre-from-repo* since that branch is still a bit away from building and I think we want to land this so that progress can be made on the Jenkins relocation.

File: chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/jenkins_steps/main [hunk 1]
```diff
@@ -10,10 +10,9 @@ check_for_autopass
 export CLUSTER_CONFIG_TEMPLATE=${CLUSTER_CONFIG_TEMPLATE:-"$CHROMA_DIR/chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/shared_storage_configuration_cluster_cfg.json"}
 
 cd $WORKSPACE
-set +x  # DONT REMOVE/COMMENT or you will risk exposing the jenkins-pull api token in the console logs.
-echo "+ curl -f -k -O -u $JENKINS_USER:********** \"$JOB_URL/$ARCHIVE_NAME\""
-curl -f -k -O -u $JENKINS_USER:$JENKINS_PULL "$JOB_URL/$ARCHIVE_NAME"
-set -x
+
+echo "+ curl -f -k -O \"$JOB_URL/$ARCHIVE_NAME\""
+curl -f -k -O "$JOB_URL/$ARCHIVE_NAME"///
```
Comment (brianjmurrell) [inline chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/jenkins_steps/main]: I could have sworn that I mentioned removing these trailing slashes in a previous review.

There were other comments IIRC.

**Comment by jgrund** [comment]
Yeah, this patch is effectively dead. I'm keeping it around just for public Jenkins testing.

**Comment by brianjmurrell** [comment]
> Yeah, this patch is effectively dead. I'm keeping it around just for public Jenkins testing.

So did the changes land on master elsewhere?  If so, why do we need to keep this one around?  Can't public Jenkins just test on master?

**Pull request closed without merging**